downloader/src/main/kotlin/vcs/GitWorkingTree.kt

```diff
@@ -77,7 +77,8 @@ open class GitWorkingTree(
                         }
                     } else {
                         paths += path
-                        walk.repository.use { submoduleRepo ->
+                        walk.repository.use {
+                            submoduleRepo ->
```

Comment:
That's not what I mean. I meant the code should be formatted like

```
                        paths += path

                        walk.repository.use { submoduleRepo ->
                            listSubmodules(path, submoduleRepo, paths)
                        }
```

And please squash this change with the first commit instead of adding a new one.

Reply:
done